### PR TITLE
I12 super admin page

### LINKED ISSUE #12: Create Admin Page
## Basic Information
This is the page will be navigated to after an admin user has logged in
- Create Tournament Button
- Create new instiution
- Create new venue
- Create new Adjudicator

- Ability to manage tournaments. i.e. setup tournament & edit tournament details

## Other Information
Related to issue #13

### PR DESCRIPTION
**Pull request by daviid-wang**
## Change Summary
Created the page similar to the figma design
Components include: AdminButton and ProfileInfo
Pages: SuperAdminPage

### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [x] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information
**[Is there anything in particular in the review that I should be aware of?]**

# Related issue

- Resolve #12

### CODE REVIEW

File: components/SuperAdminPage/ProfileInfo.vue [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<template>
+    <div class="w-2/3">
+        <div class="text-6xl">{{username}}</div>
+        <div class="ml-8 text-xl">{{role}}</div>
+        <hr class="w-full mt-8 mb-6 border-black border-solid">
+    </div>
+
+    <!-- <div class="flex justify-center"> -->
+        
+    <!-- </div> -->
```
Comment (ChantelleKerr) [inline components/SuperAdminPage/ProfileInfo.vue:17]: These comments can be removed :)

File: pages/superAdminPage.vue [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    import { UserIcon } from "@heroicons/vue/24/outline";
+    import { TrophyIcon } from "@heroicons/vue/24/outline";
+    import { BuildingLibraryIcon } from "@heroicons/vue/24/outline";
+    import { ScaleIcon } from "@heroicons/vue/24/outline";
+    import { UserGroupIcon } from "@heroicons/vue/24/outline";
+    import { MapPinIcon } from "@heroicons/vue/24/outline";
+    import { TableCellsIcon } from "@heroicons/vue/24/outline";
+    import { UsersIcon } from "@heroicons/vue/24/outline";
+    import { ClipboardIcon } from "@heroicons/vue/24/outline";
+    import { UserPlusIcon } from "@heroicons/vue/24/outline";
```
Comment (ChantelleKerr) [inline pages/superAdminPage.vue:13]: You can import all these on the same line { UserIcon, TrophyIcon, .... , UserPlusIcon }

File: pages/superAdminPage.vue [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+</script>
+
+<template>
+    <div class="flex items-center justify-center">
+        <UserIcon class="w-28 h-28" />
+        <ProfileInfo username="Carmen Leong" role="Super Admin"/>
```
Comment (ChantelleKerr) [inline pages/superAdminPage.vue:19]: Change Role to just "Admin"

File: pages/superAdminPage.vue [hunk 4]
```diff
@@ -0,0 +1,34 @@
+<script setup>
+    import AdminButton from '~/components/SuperAdminPage/AdminButton.vue';
+    import ProfileInfo from '~/components/SuperAdminPage/ProfileInfo.vue';
```
Comment (ChantelleKerr) [inline pages/superAdminPage.vue:3]: Change the file name to just admin.vue

**Review by ChantelleKerr — CHANGES_REQUESTED** [review]
Great job! It looks really good and I love that you used different components

File: components/SuperAdminPage/ProfileInfo.vue [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<script setup>
+        const {props} = defineProps({
+        username: String,
+        role: String,
+    })
+</script>
+
+<template>
+    <div class="w-2/3">
```
Comment (rezbyte) [inline components/SuperAdminPage/ProfileInfo.vue:9]:
```suggestion
    <section class="w-2/3">
```
Semantic HTML over just `div`s is better for accessibility.

File: components/SuperAdminPage/ProfileInfo.vue [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+        <div class="text-6xl">{{username}}</div>
+        <div class="ml-8 text-xl">{{role}}</div>
+        <hr class="w-full mt-8 mb-6 border-black border-solid">
+    </div>
```
Comment (rezbyte) [inline components/SuperAdminPage/ProfileInfo.vue:13]:
```suggestion
    </section>
```

File: components/SuperAdminPage/ProfileInfo.vue [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+        const {props} = defineProps({
+        username: String,
+        role: String,
+    })
+</script>
+
+<template>
+    <div class="w-2/3">
+        <div class="text-6xl">{{username}}</div>
+        <div class="ml-8 text-xl">{{role}}</div>
```
Comment (rezbyte) [inline components/SuperAdminPage/ProfileInfo.vue:11]:
```suggestion
        <p class="ml-8 text-xl">{{role}}</p>
```

File: pages/superAdminPage.vue [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+    import { UserGroupIcon } from "@heroicons/vue/24/outline";
+    import { MapPinIcon } from "@heroicons/vue/24/outline";
+    import { TableCellsIcon } from "@heroicons/vue/24/outline";
+    import { UsersIcon } from "@heroicons/vue/24/outline";
+    import { ClipboardIcon } from "@heroicons/vue/24/outline";
+    import { UserPlusIcon } from "@heroicons/vue/24/outline";
+</script>
+
+<template>
+    <div class="flex items-center justify-center">
```
Comment (rezbyte) [inline pages/superAdminPage.vue:17]: I think all of this would be better placed inside the `ProfileInfo` component.

**Review by rezbyte — CHANGES_REQUESTED** [review]
Nice work so far! :slightly_smiling_face: 
I'm getting some Vue errors in the terminal when visiting the super admin page.
There's also some linting errors that the workflow didn't pick up.
I'll have to fix the workflow. (Update: #72)
Don't forget to use the automatically generated branches rather than creating your own.
Other than that, keep it up! :+1:

**Comment by daviid-wang** [comment]
I'll have a look at these tonight, and yeah I forgot to do linting oops

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Review by ChantelleKerr — APPROVED** [review]
Looks good! Great job :)

**Comment by rezbyte** [comment]
> Looks good! Great job :)

I'll just merge this branch with main to check for linting problems first.

**Review by rezbyte — APPROVED** [review]
Well done! :fireworks: 
Merge when you're ready.

**Pull request merged**